hrrecruitment/tests/phpunit/CRM/HRRecruitment/BAO/HRVacancyTest.php

```diff
@@ -18,12 +18,6 @@ public function setUpHeadless() {
       ->apply();
   }
 
-  function setUp() {
-  }
-
-  function teardown() {
-  }
-
   protected static function _populateDB($perClass = FALSE, &$object = NULL) {
```

Comment:
oh sorry ... no problem I will remove them in a one single commit with other fixes .. thanks